prow/plugins.yaml

```diff
@@ -61,6 +61,12 @@ approve:
   - kubernetes/test-infra
   implicit_self_approve: true
   lgtm_acts_as_approve: true
+# Disabling implicit self-approval for a given repo in the
```

Comment:
We need to fix this to digest the entire config and resolve to what is appropriate. Right now these "overrides" are possible since we assume a repo is only configured by one entry in config, and we pick the first one we find. /cc @cjwagner

Reply:
Agreed that having an explicit order of resolution would be desirable.

On the issue of implicit self-approval, why is this the default?

Reply:
Arg, looks like my assumption was incorrect as per the linked test failure (detects potential conflict).  Any suggestions for how to achieve the desired behavior (self approval disabled for fedv2)?